pom.xml

```diff
@@ -230,8 +230,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
Ok, I switched back to Java 6. I wasn't brave enough to fix compilation for all API changes between `1.612` and `1.625.3`.